src/OpenTelemetry.Api/README.md

```diff
@@ -270,13 +270,17 @@ OpenTelemetry samplers chose not to sample this activity.
     activityLinks.Add(new ActivityLink(linkedContext2));
 
     var activity = activitySource.StartActivity(
-        "ActivityName",
+        "ActivityWithLinks",
         ActivityKind.Server,
-        "00-0af7651916cd43dd8448eb211c80319c-b7ad6b7169203331-01",
+        default(ActivityContext), // this creates Activity without parent
         initialTags,
         activityLinks);
     ```
 
+    In case you have a parent and it's remote specify `traceaparent` string
```

Comment:
I don't know if `traceparent` is a familiar term for the reader, consider adding a reference link.